js/os_test.ts

```diff
@@ -204,3 +204,22 @@ testPerm({ write: false }, function mkdDirSyncPerm() {
   assertEqual(err.name, "deno.PermissionDenied");
 });
 
+testPerm ({ write: true }, function chmodSync() {
+  const path = "/README.md";
+    const mode = "777";
+    deno.chmodSync(path, mode);
+});
```

Comment:
It would be better to create a temp directory and use that for the chmod target.